### DISCUSSION TITLE
New visits count as pageview?

### DISCUSSION THREAD

**Discussion by dotgodly** [plausible/analytics#3604] [Anything Else]
When closing the tab and reopening it a time later, it counts as a pageview not a visit, even if the utm_source is different. Is this expected behavior?

I would expect it to count as a new visit (albeit not unique), because it currently throws off the visit duration.

**Comment by metmarkosaric** [comment]
Hi @dotgodly! The existing visit (session) needs to end for it to count as a new visit. A visit is started when a visitor first lands on your page and ends when no action is taken on your site for 30 minutes. We have all the different metrics definitions here https://plausible.io/docs/metrics-definitions

**Reply by dotgodly** [reply]
Thank you.
